### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by fnt90**
## Changes Proposed
- Import all css files to styles.css
- Rename App.js to App.jsx
- Import Google font: Oxygen
- Create 3 color variables in _variables.css
- Import script for FontAwesome icons
- Add code for star rating
- Add Munchbox favicon

### CODE REVIEW

**Review by sailathatammana — COMMENTED** [review]
Good

File: frontend/src/components/star-rating.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+export default function StarRating() {
+    const [rating, setRating] = useState(0);
+    const [hover, setHover] = useState(0);
```
Comment (supernova4888) [inline frontend/src/components/star-rating.jsx:5]: test

File: frontend/src/components/star-rating.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <div>
+            <div className="star-rating">
+                {[...Array(5)].map((star, index) => {
+                    index += 1;
+                    return (
+                        <button
+                            type="button"
+                            key={index}
+                            className={index <= (hover || rating) ? "on" : "off"}
+                            onClick={() => setRating(index)}
```
Comment (supernova4888) [inline frontend/src/components/star-rating.jsx:16]: so wrong

**Comment by supernova4888** [comment]
lookin' good

File: frontend/public/index.html [hunk 3]
```diff
@@ -9,11 +9,13 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+    <!-- Below script to use Font Awesome icons -->
```
Comment (supernova4888) [inline frontend/public/index.html:12]: dfsffsd

**Pull request merged**